content/en/docs/Concepts/crds/bundleIndexImage.md

```diff
@@ -0,0 +1,97 @@
+```
+
+The `make bundle` command also creates a Dockerfile (bundle.Dockerfile) which is used to build a "bundle image." The bundle image is an [OCI](https://github.com/opencontainers/image-spec) image that essentially holds the generated on-disk bundle manifest and metadata files.`
+
+### Building the Operator Index Image
+
+Another OLM concept is the "index image", which is like a catalog of Operators. This container image acts to serve an application programming interface (API) which describes information about your sample operator and others in the catalog. An index image, based on the Operator Bundle Format, is a containerized snapshot of a catalog. It is an immutable artifact that contains the database of pointers to a set of Operator manifest content. A catalog can reference an index image to source its content for OLM on the cluster. 
+
+#### Creating an index image:
+
+
+1. The index image includes information from your bundle image by running an operator-registry command, opm, as follows:
+```
+opm index add --bundles <registry>/<username>/sample-operator-bundle:vX.X.X --tag <registry>/<username>/sample-operator-index:vX.X.X --binary-image <registry_base_image>
+```
+- Comma-separated list of bundle images to add to the index.
+- The image tag that you want the index image to have.
+- Optional: An alternative registry base image to use for serving the catalog.
+
+2. Push the index image to a registry:
+```
+docker push <registry>/<username>/sample-operator-index:vX.X.X
+```
+
+The index image holds a sqlite database with bundle definitions, it also runs a grpc service when the image is executed. The grpc service lets consumers query the sqlite database for information about the operators this index contains.
+
+You can download the opm command from the [operator-registry](https://github.com/operator-framework/operator-registry) repository.
+
+#### Creating a catalog from an index image
+
+Create a CatalogSource object that references your index image. Modify the following to your specifications and save it as a catalogSource.yaml file:
+
+```yaml
+kind: CatalogSource
+metadata:
+  name: sample-operator
+  namespace: operators
+spec:
+  sourceType: grpc
+  image: <registry>:<port>/<namespace>/sample-operator-index:vX.X.X
+  displayName: My Operator Catalog
+  publisher: <publisher_name> 
+  updateStrategy:
+    registryPoll: 
+      interval: 30m
+```
+
+- Specify your index image.
+- Specify your name or an organization name publishing the catalog.
+- Catalog sources can automatically check for new versions to keep up to date.
+
+Now we have to apply this yaml file to the kubernetes cluster:
+
+```
+kubectl apply -f catalogSource.yaml
+```
+
+​This CatalogSource is created in an existing namespace, created by OLM, named operators; on OpenShift clusters, this namespace is called openshift-operators. When you create the CatalogSource, it causes the index image to be executed as a Pod. You can view it as follows:
```

Comment:
Please update and replace commas so it reads clear such as:
This CatalogSource is created in an existing namespace by OLM named operators on OpenShift clusters. This namespace is called openshift-operators. When you create the CatalogSource, it causes the index image to be executed as a Pod. You can view it as follows: